example/src/main/java/com/spruceid/mobilesdkexample/wallet/WalletHomeView.kt

```diff
@@ -86,33 +94,72 @@ fun WalletHomeHeader(navController: NavController) {
 }
 
 @Composable
-fun WalletHomeBody(rawCredentialsViewModel: IRawCredentialsViewModel) {
+fun WalletHomeBody(
+    rawCredentialsViewModel: IRawCredentialsViewModel,
+    navController: NavController
+) {
     val scope = rememberCoroutineScope()
 
     val rawCredentials by rawCredentialsViewModel.rawCredentials.collectAsState()
 
     if(rawCredentials.isNotEmpty()) {
-        LazyColumn(
-            Modifier
-                .fillMaxWidth()
-                .padding(top = 20.dp)
-        ) {
-            items(rawCredentials) { rawCredential ->
-                AchievementCredentialItem(
-                    rawCredential.rawCredential,
-                    onDelete = {
-                        scope.launch {
-                            rawCredentialsViewModel.deleteRawCredential(id = rawCredential.id)
+        Box(modifier = Modifier.fillMaxSize()) {
+            LazyColumn(
+                Modifier
+                    .fillMaxWidth()
+                    .padding(top = 20.dp)
+            ) {
+                items(rawCredentials) { rawCredential ->
+                    AchievementCredentialItem(
+                        rawCredential.rawCredential,
+                        onDelete = {
+                            scope.launch {
+                                rawCredentialsViewModel.deleteRawCredential(id = rawCredential.id)
+                            }
                         }
+                    ).component()
+                }
+    //        item {
+    //            vcs.map { vc ->
+    //                GenericCredentialListItems(vc = vc)
+    //            }
+    //            ShareableCredentialListItems(mdocBase64 = mdocBase64)
+    //        }
+            }
+
+            Box(
+                modifier = Modifier.fillMaxSize(),
+                contentAlignment = Alignment.BottomCenter
+            ) {
+                Button(
+                    onClick = {
+                        navController.navigate(Screen.OID4VPScreen.route)
+                    },
+                    modifier = Modifier.fillMaxWidth(),
+                    colors = ButtonDefaults.buttonColors(
+                        containerColor = Color(0xFF488CF4),
```

Comment:
you must create a color variable, the same way we have for `CTAButtonGreen`. You can call it `CTAButtonBlue` and create it on `Color.kt` and `colors.xml`